Gemfile

```diff
@@ -48,11 +48,11 @@ else
   gem 'gds-sso', '10.0.0'
 end
 
-gem 'govuk_admin_template', '1.4.0'
+gem 'govuk_admin_template', '2.3.1'
```

Comment:
If bumping 1.x to 2.x, please remove GA code, see https://github.com/alphagov/travel-advice-publisher/pull/81 as an example